cmodelres/CMakeLists.txt

```diff
@@ -42,6 +42,31 @@ else()
     message(WARNING "Failed to find sea. See -DSEA_PATH.")
 endif()
 
+# Locates KLEE, as required for testing.
```

Comment:
I anticipate some conflicts with `cmodel-dev` now. We probably want to pull in that branch first.

Reply:
Ah, just noticed that you pulled https://github.com/ScottWe/solidity-to-cmodel/pull/10 which has very slight overlap: I think we can now drop `set(CMAKE_C_COMPILER clang)` in this branch.